src/protocol.ts

```diff
@@ -75,3 +75,9 @@ export namespace MetalsWindowStateDidChange {
 export interface MetalsWindowStateDidChangeParams {
   focused: boolean;
 }
+
+export interface MetalsNewScalaFileParams {
+  directory?: string;
+  name: string;
+  kind: string;
```

Comment:
can this kind be more specific?

```suggestion
  kind: 'class' | 'object' | 'trait' | 'package-object' | 'worksheet' ;
```